COMRS-121 Load location hierarchy

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COMRS-121

Commit:
COMRS-60 Add cameroon locations

tmvumbi2 committed:
COMRS-121 Add config for address hierarchy and entries

Adds configurations to load the address structure to be used for the Cameroon project and address entries. These configuration files are read by the addresshierarchy module version 2.12 (the default version shipped with Bahmni is 2.9; it needs to be updated) which automatically imports the address template and entries.

tmvumbi2 committed:
COMRS-121 Add i18n to address config file

Replaces name mappings in the address configuration by i18n keys (which are defined in 'config/openmrs/i18n/registration/locale_[en/fr].json').

rubailly reviewed (changes requested):
Let's follow the configuration that the default Address Hierarchy module that comes with bahmni can understand. I think using this configuration is what caused the import of locations to fail and forced you to upgrade the Address Hierarchy module first.

[This page](https://bahmni.atlassian.net/wiki/spaces/BAH/pages/37945505/Configure+Address+Hierarchy) can be helpful.

rubailly reviewed (changes requested):
Why are the entries all in capital case? 
...and I would try to avoid using a comma as much as possible. -Use a pipe as separator instead.
It is common to have address names with a comma in the actual name and this will probably result in some errors during import. 

...see for example the following line in this entries file:

CAMEROUN,NORD-OUEST,MEZAM,BAMENDA 2,BAMENDA 2 ,BAMENDA TOWN ,"ATU-AZIRE (META QUATER,BIG MANKON) "

tmvumbi2 committed:
COMRS-121 Use pipe as delimiter for entries.csv

This commit:
- uses the pipe as a field delimiter in the csv file instead of comma; the reason for this is that some address entries do have a comma in their value which interferes with the resulting csv;
- updates the mapping: uses cityVillage to map the city level: this is the closest 'natural mapping' to an existing address level in OpenMRS. The commit also uses the smallest index (i.e. address1) on the smallest hierarchy level (i.e quartier); this way of mapping is suggested in the documentation and followed by several implementations (see https://wiki.openmrs.org/display/docs/Administering+Address+Templates)

tmvumbi2 committed:
COMRS-121 Update the translation of 'Quartier'

Uses 'Area' instead of 'Suburb': area is a better translation of quartier

Commit:
Revert "COMRS-60 Add cameroon locations"

This reverts commit 379008c3ad7280c1220fcd259463285beab8b940.
